### DISCUSSION TITLE
GDAS Data is Not Present in the LIS Data Portal

### DISCUSSION THREAD

**Discussion by srikar-iist** [NASA-LIS/LISF#1261] [Q&A] [answered]
Hi All,

The LIS data portal no longer has the GDAS data for Running LIS. I believe it has been removed / shifted to another website.

Kindly advise on how to access / download the data.

Thanks in advance

**Comment by karsenau** [comment, marked as answer]
Hi @srikar-iist:

Thank you for reporting this issue.  The issue looks now to be resolved, so please try your downloads again.
